tests/framework/cli/test_starters.py

```diff
@@ -200,33 +195,6 @@ def test_fail_if_dir_exists(self, fake_kedro_cli):
         assert result.exit_code != 0
         assert "directory already exists" in result.output
 
-    @pytest.mark.parametrize(
-        "repo_name", [".repo\nvalid", "re!po\nvalid", "-repo\nvalid", "repo-\nvalid"]
```

Comment:
I think we still need these test cases, it would catch the bug that I found.

Reply:
I have created a new test `test_invalid_project_name` for this case